src/cephadm/cephadm

```diff
@@ -3679,6 +3672,18 @@ def prepare_ssh(
             cli(['orch', 'apply', t])
 
 
+def enable_cephadm_mgr_module(
+    cli: Callable, wait_for_mgr_restart: Callable
+) -> None:
+
+    # enable cephadm mgr module
```

Comment:
this comment is not necessary, i think. as it practically repeats the following logging message.